repositorytools/cli/common.py

```diff
@@ -55,5 +58,5 @@ def run(self, args=None):
         return args_namespace.func(args_namespace)
 
     def __call__(self, *args):
-        self.run(args)
+        self.run(*args)
```

Comment:
Yes, that was expected. I left this commit so that this `adding_version_flag` branch could be merged once you included the `*args` fix.